### PR TITLE
Allow Vultr to set MTU and use as-is configs

### PR DESCRIPTION
**Pull request by eb3095**
## Proposed Commit Message
Allow Vultr to set MTU, accept-ra, routes, and use as-is configs

```
Add MTU, accept-ra, routes, options and a direct way to provide intact 
cloud configs for networking opposed to relying on configurations that 
may need changed often.
```

## Additional Context
Our VPC project required a few more things than I was aware of . Sorry for the multiple PR's

## Checklist:
 - [ X ] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/hacking.html)
 - [ X ] I have updated or added any unit tests accordingly
 - [ X ] I have updated or added any documentation accordingly

### CODE REVIEW

File: cloudinit/sources/helpers/vultr.py [hunk 1]
```diff
@@ -202,7 +220,30 @@ def generate_private_network_interface(interface):
...
+        netcfg['accept-ra'] = interface['accept-ra']
+
+    if "routes" in interface:
+        netcfg['subnets'][0]['routes'] = interface['routes']
+
     return netcfg
 
 
+# Make required adjustments to the network configs provided
+def do_network_interfaces(interfaces):
```
Comment (TheRealFalcon) [inline cloudinit/sources/helpers/vultr.py:237]: Can we make this name more descriptive? `add_interface_names` or something similar?

**Review by TheRealFalcon — COMMENTED** [review]
Looks good! My only comment is a minor naming issue.

File: cloudinit/sources/helpers/vultr.py (hunk 1 above)
Comment (eb3095) [reply cloudinit/sources/helpers/vultr.py]: Sure can. Was having a hard time naming that one as I wasn't sure if there was going to be anything else. At this time, likely not! Changed.

**Review by TheRealFalcon — APPROVED** [review]
Thanks!

**Comment by TheRealFalcon** [comment]
FYI, travis is currently broken on main, so I won't be landing this PR until it is fixed. This has nothing to do with your PR. It should get fixed with #1043

**Comment by eb3095** [comment]
Yeah I saw that one last night. That's fine. I'll keep an eye on it and merge in main again when its in.

**Pull request merged**